src/ODE.jl

```diff
@@ -456,9 +506,14 @@ function ode23s(F, y0, tspan; reltol = 1.0e-5, abstol = 1.0e-8,
     tfinal = tspan[end]
     tdir = sign(tfinal - t)
 
-    hmax = abs(tfinal - t)/10
-    hmin = abs(tfinal - t)/1e18
-    h = tdir*abs(tfinal - t)/100  # initial step size
+    hmax = maxstep
```

Comment:
Is there a reason not to just use the original variables? Why do we need to alias them?